### PR TITLE
Write readme

### LINKED ISSUE #2: Write a README
Write a proper README.md for the repository

### PR DESCRIPTION
**Pull request by EltayebAhmed**
Added a brief introduction and requirments to the Readme.
Instructions on how to actually start mantidplot are in another branch along with the actual scripts

**To test:**

Read the `readme.md`

Fixes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+To check the version of Matplotlib on your system open the `MantidPlot` application and in the script interpreter at the bottom of the window execute ` import matplotlib; print matplotlib.__version__`
+The output should look like
+```
+In [1]: import matplotlib; print matplotlib.__version__
+1.5.1
+
+In [2]:
+```
+If the version that appears in the interpreter is less that  `1.5.1` then you must upgrade it by executing the following command in the MantidPlot script interpreter
+ `%run -m easy_install -- --user "matplotlib>=1.5.1"`
```
Comment (quantumsteve) [inline README.md]: I'm concerned that user installs of matplotlib are going to generate a future support headache. 

- Our RHEL build servers don't test this configuration. 
- In the future, a newer system version of Matplotlib (@martyngigg mentioned wanting a quick transition to Matplotlib 2.0) will be hidden.

May we redirect our energies towards building a working matplotlib 1.5+ rpm of RHEL7? Can we advertise a more isolated solution, such installing mantid-framework in an Anaconda environment, here?

@stuartcampbell @peterfpeterson

Comment (peterfpeterson) [inline README.md]: Do you mean a [conda recipe](https://github.com/mantidproject/conda-recipes) which is intended to be run after every (successful) nightly build and push its results to [anaconda](https://anaconda.org/mantid/mantid-framework)?

Comment (quantumsteve) [inline README.md]: How else might we address the issues raised above?

**Comment by martyngigg** [comment]
Closing as stale.

**Pull request closed without merging**